packaging.sbt

```diff
@@ -1,9 +0,0 @@
-import NativePackagerKeys._
```

Comment:
I removed this file since we deploy to heroku and don't need this stuff right now. Also the API changed.